README.md

```diff
@@ -60,3 +59,13 @@ app.use(busboy({
 }));
 
 ```
+
+## Troubleshooting
+
+### 'TypeError: Cannot call method 'on' of undefined'
+
+If you find that `req.busboy` is not defined in your code when you expect it to be, check that the following conditions are not. If they are not, `req.busboy` won't be defined:
+
+ 1. The request method is not GET or HEAD
+ 2. The Content-Type header specifies  that is "application/x-www-formurlencoded" or starts with "multipart/*"
+ 3. The Content-Length header is defined or chunked transfer encoding is in use.
```

Comment:
The 3rd note can be removed since all web requests are going to either have chunked encoding or `Content-Length`. Have you seen other scenarios out there?